src/Digdir.Domain.Dialogporten.ChangeDataCapture/ChangeDataCapture/Checkpoints/CheckpointSyncronizer.cs

```diff
@@ -26,7 +24,7 @@ internal sealed class CheckpointSyncronizer : BackgroundService, IAsyncDisposabl
         _snapshotCache = cache ?? throw new ArgumentNullException(nameof(cache));
         _snapshotRepository = snapshotRepository ?? throw new ArgumentNullException(nameof(snapshotRepository));
         _logger = logger ?? throw new ArgumentNullException(nameof(logger));
-        _optionsChangeSubscription = options?.OnChange(UpdateOptions) ?? throw new ArgumentNullException(nameof(options));
+        _optionsChangeSubscription = options.OnChange(UpdateOptions) ?? throw new ArgumentNullException(nameof(options));
```

Comment:
Dersom options er null vil vi få en NullReferenceException i stede for en ArgumentNullException. Den første har tradisjonelt nok info til å finne ut ballpark av hvor ting tryna, men ikke nok info til å finne ut akkurat hva som tryna. ArgumentNullException er mange ganger bedre enn NullReferenceException 🙂